xbmc/GUILargeTextureManager.cpp

```diff
@@ -57,27 +57,33 @@ bool CImageLoader::DoWork()
   else
     loadPath = texturePath;
 
-  if (m_use_cache && loadPath.empty())
-  {
-    // not in our texture cache, so try and load directly and then cache the result
-    loadPath = CTextureCache::Get().CacheImage(texturePath, &m_texture);
-    if (m_texture)
-      return true; // we're done
-  }
   if (!m_use_cache || !loadPath.empty())
   {
```

Comment:
@mkortstiege : Btw. you see the logic in the above if-statement above? Seems there's no point doing this block if loadPath is empty (so with it being empty and m_use_cache = 0 we would enter this block)?